### DISCUSSION TITLE
subscribe functionality in python sdk?

### DISCUSSION THREAD

**Discussion by Ananya2001-an** [appwrite/appwrite#5596] [Q&A] [answered]
Hey folks! I was just wondering if there is a way to subscribe to a collection via python sdk for realtime updates. On the official docs for realtime they have only given an example for the web sdk. I am kinda confused as to how to implement the same in python. Thanks in advance :)

**Comment by byawitz** [comment, marked as answer]
Hey,
Realtime doesn't support the server-side SDKs.

https://appwrite.io/docs/realtime#known-limitations

**Reply by Ananya2001-an** [reply]
ahh okay thanks for the response @byawitz I hope that will be soon added in the future... :)
